src/test/java/technology/semi/weaviate/integration/client/graphql/ClientGraphQLTest.java

```diff
@@ -1,30 +1,24 @@
 package technology.semi.weaviate.integration.client.graphql;
 
 import java.io.File;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
+
 import org.junit.Before;
 import org.junit.ClassRule;
 import org.junit.Test;
 import org.testcontainers.containers.DockerComposeContainer;
 import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.shaded.org.apache.commons.lang.ArrayUtils;
 import technology.semi.weaviate.client.Config;
 import technology.semi.weaviate.client.WeaviateClient;
 import technology.semi.weaviate.client.base.Result;
 import technology.semi.weaviate.client.v1.batch.model.ObjectGetResponse;
 import technology.semi.weaviate.client.v1.data.model.WeaviateObject;
 import technology.semi.weaviate.client.v1.graphql.model.ExploreFields;
 import technology.semi.weaviate.client.v1.graphql.model.GraphQLResponse;
-import technology.semi.weaviate.client.v1.graphql.query.argument.GroupArgument;
-import technology.semi.weaviate.client.v1.graphql.query.argument.GroupType;
+import technology.semi.weaviate.client.v1.graphql.query.argument.*;
```

Comment:
could you setup your IDE not to collapse imports?

Reply:
yeah, good point, please organize imports